### ISSUE TITLE
geo look ip 没有显示全部的 GeoIP codes

### ISSUE THREAD

**Issue by peeweep** [MetaCubeX/geo#1]
geoip.dat 来自 https://github.com/peeweep/geoip/releases/tag/202401111336
`223.119.50.147` 同时在  `GEOIP,CMI (223.118.0.0/15)` 和 `GEOIP,CN (223.64.0.0/10)` 中，使用其他工具比如 clash-meta 两个都可以匹配到，但是 geo look 只返回了 CN

实际结果
```
$ geo look 223.119.50.147 
🔎Querying from [/home/peeweep/.geo/geoip.dat]
🎉Query finished in 46.96831ms!
Total 1 results (GeoIP codes):
  CN %  
```

期望结果: 
```
$ geo look 223.119.50.147 
🔎Querying from [/home/peeweep/.geo/geoip.dat]
🎉Query finished in 46.96831ms!
Total 2 results (GeoIP codes):
  CMI CN %  
```

**Comment by H1JK** [comment]
This may be related to some issues with database format conversion, since we convert all v2ray formats to internal types for lookups. And for GeoIP it is converted to Meta-geoip for now.

I have done some research and will try to fix it.

**Comment by H1JK** [comment]
Fixed in 474a473. Please try the latest build or build it from source.

**Comment by peeweep** [comment]
Thanks!
